src/manifest.c

```diff
@@ -1170,7 +1170,7 @@ struct file *search_bundle_in_manifest(struct manifest *manifest, const char *bu
 	struct list *iter = NULL;
 	struct file *file;
 
-	iter = manifest->manifests;
+	iter = list_head(manifest->manifests);
```

Comment:
I agree that the list_head() is needed here, but this function is not called by verify_fix_path()... Can you update the commit description to reflect that this is fixing a general issue, not just for verify_fix_path()?